src/api/java/appeng/api/features/IP2PTunnelRegistry.java

```diff
@@ -47,6 +48,7 @@
 	 */
 	void addNewAttunement( @Nonnull ItemStack trigger, @Nullable TunnelType type );
 	void addNewAttunement( @Nonnull String ModId, @Nullable TunnelType type );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00117)